metadatastore/conf.py

```diff
@@ -3,18 +3,29 @@
 import logging
 
 logger = logging.getLogger(__name__)
-filename = os.path.join(os.path.expanduser('~'), '.config', 'metadatastore',
-                        'connection.yml')
-if os.path.isfile(filename):
-    with open(filename) as f:
-        connection_config = yaml.load(f)
-    logger.debug("Using db connection specified in config file. \n%r",
-                 connection_config)
-else:
-    connection_config = {
-        'database': 'test',
-        'host': "localhost",
-        'port': 27017,
-        }
```

Comment:
If mds/fs are not going to install config files by default, we should have examples of what they should look like here: http://nsls-ii.github.io/install.html, right?